betel/categories_enum.py

```diff
@@ -0,0 +1,59 @@
+import enum
+
+
+class Categories(enum.Enum):
```

Comment:
I feel it might be easier to just to define the categories as a list, if they are going to be used to enumerate Play store listings to scrape the top apps e.g. https://play.google.com/store/apps/category/GAME_CASUAL and being used as class labels.

But depends how you plan to use them, in other words, +1 to Momo's comment below.

Reply:
The mappings represent the expected strings when retrieving the category from the app's page. Scraping the listings did not generate enough apps, so we dropped it.